extide/gradle/src/org/netbeans/modules/gradle/api/GradleBaseProject.java

```diff
@@ -319,6 +335,9 @@ GradleConfiguration createConfiguration(String name) {
      *         {@code null} for non-Gradle projects.
      */
     public static GradleBaseProject get(Project project) {
+        if (project == null) {
```

Comment:
Is this needed? NbGradleProject.get() should return null on a null project.

Reply:
addressed in 2db82c7